src/components/ControlBar.js

```diff
@@ -13,45 +13,45 @@ function ControlBar (): React.MixedElement {
   const buttonClicked = (e: SyntheticEvent<HTMLButtonElement>): void => {
     const modalType: string = e.currentTarget.name;
 
-    if (modalType === 'teamConfig') {
+    if (modalType === "teamConfig") {
```

Comment:
just for clarity I think in general our team does single quotes for strings, it's only double quotes for react prop strings, though this isn't something anyone would really enforce since double quotes is perfectly fine too

Reply:
or if you want to do possession aka "long's code" then you'll def want to use double quotes